starter-code/lodash.js

```diff
@@ -0,0 +1,55 @@
+
+
+
+var opinions = [ "This is the best job ever",
+                 "Satisfied",
+                 "At least I get paid",
+                 "I'm looking for another job",
+                 "I don't want to answer"];
+
+var tabOpinion = [];
+
+
+function pickOpinion  () {
+  var indexOpinion = Math.floor (Math.random()*opinions.length);
+  return opinions[indexOpinion];
+}
+
+pickOpinion ();
+
+var1 = _.sample(opinions);
```

Comment:
don't forget the var in front of the variable names. Also never ever write `var1` again 😆  That name is so unexplicit!